DigitalRain/DigitalRainConfig.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+using System.Diagnostics;
+using System.IO;
+
+namespace DigitalRain
+{
+    public class ColumnNumberPickerConfig
+    {
+        [JsonProperty(Required = Required.Always)]
+        public string type;
+
+        [JsonProperty(Required = Required.Always)]
+        public int columnCount;
+
+        public int lowWaterMark = 1;
+    }
+
+    public class DigitalRainConfig
+    {
+        public ColumnNumberPickerConfig columnNumberPicker;
+    }
+
+    public class ConfigReader
+    {
+        public DigitalRainConfig ReadConfig(string filename)
```

Comment:
For now, could this function live inside `DigitalRainConfig` for now? Having a whole `ConfigReader` seems a bit much and it will return the DigitalRainConfig so it seems like more complexity.

Reply:
Not sure.  `DigitalRainConfig` is special because the JSON library looks at it to determine how to serialize/deserialize JSON.  IDK what happens if you put a method on it.  I almost put it as a method on the `DigitalRainGame`, but then you'd have to import the JSON reader library there and that seems like clutter.  Then I wanted a free function but I assume you can't do that in C# because it sucks.